charts/mysql/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,46 @@
+{{- /*
```

Comment:
Are we going to support this serviceMonitor? We do not use serviceMonitors anywhere else in our chart. I feel like we should remove all mentions of serviceMonitor.